test/modules/aggregation-layers/index.js

```diff
@@ -22,10 +22,12 @@ import './contour-layer/contour-layer.spec';
 import './contour-layer/marching-squares.spec';
 import './gpu-grid-layer/gpu-grid-cell-layer-vertex.spec';
 import './gpu-grid-layer/gpu-grid-layer.spec';
+import './gpu-grid-layer/gpu-grid-cell-layer.spec';
 import './cpu-grid-layer/cpu-grid-layer.spec';
 import './grid-aggregator.spec';
 import './hexagon-layer.spec';
 import './hexagon-aggregator.spec';
 import './grid-layer.spec';
 import './screen-grid-layer.spec';
 import './utils/scale-utils.spec';
+import './utils/aggregation-operation-utils.spec';
```

Comment:
This needs to be removed from `test/browser.js`

Reply:
Any specific reason? Its testing pure JS code only.

Reply:
`test/browser.js` already imports `modules/index.js` so this ends up being imported twice.